packages/cln_plugin/lib/src/rpcmethods/manifest/types/feature.dart

```diff
@@ -0,0 +1,31 @@
+class Feature {
+  late String _node;
+  late String _channel;
+  late String _init;
+  late String _invoice;
+
+  Feature(String node, String channel, String init, String invoice) {
```

Comment:
Here with the constructor parameter, you can use the `{required String node, ..}`

This is a little bit confusing to me, can you point to the cln docs about the Feature part?

Reply:
[Here](https://lightning.readthedocs.io/PLUGINS.html#the-getmanifest-method).
Maybe I should have gone with ```FeatureBits```.
```
"featurebits": {
    "node": "D0000000",
    "channel": "D0000000",
    "init": "0E000000",
    "invoice": "00AD0000"
  },
```